editor/plugins/canvas_item_editor_plugin.cpp

```diff
@@ -2276,14 +2276,14 @@ void CanvasItemEditor::_draw_grid() {
 			real_grid_offset = grid_offset;
 		}
 
-		const Color grid_minor_color = get_color("grid_minor_color", "Editor");
+		const Color grid_color = EditorSettings::get_singleton()->get("editors/2d/grid_color");
```

Comment:
Maybe `grid_minor_color` should be removed from the theme then? It's now only used in:
```
editor/plugins/curve_editor_plugin.cpp
620:    const Color grid_color1 = get_color("grid_minor_color", "Editor");
```

Reply:
Done. I also removed grid_major_color, since its only used in few places of code.